Extension/Services/TroopActionService.cs

```diff
@@ -78,25 +78,39 @@ public static (int, int) RecruitPrisoners(PartyVM partyVm, PartyScreenLogic part
                 }
             }
 
+            var playerSelectedCharacter = partyVm.CurrentCharacter.Character;
```

Comment:
Not a massive fan of var, would prefer types be explicitly stated

Reply:
I think some cases where the class looks a little verbose var would be good. This case might be debatable. Changed var to CharacterObject for now